sns/savant-sns.json

```diff
@@ -935,7 +935,7 @@
         "CS"
       ],
       "entries": [
-        "Whenever you take a short or long rest, you can use your medical knowledge to treat one creature. That creature can heal as if it had one extra {@dice 1d4} Hit Die during this rest, or it can ignore one level of {@condition fatigue|sns} for the next four hours."
+        "Whenever you take a short or long rest, you can use your medical knowledge to treat one creature. That creature can heal as if it had one extra {@dice 1d6} Hit Die during this rest, or it can ignore one level of {@condition fatigue|sns} for the next four hours."
```

Comment:
This should have been increased already with the extra HP/healing policy of sns, just missed it originally.